### PR TITLE
Fix for crate > 2.0

### PR DESCRIPTION
**Pull request by corck**
Get driver running on Crate 2.x, maybe earlier, not sure when that change was introduced.

Mainly fixed tables query statement.
Fixed specs and a bit of refactoring.
Added byebug for better debugging in dev mode.

### CODE REVIEW

File: lib/crate_ruby/version.rb [hunk 1]
```diff
@@ -20,5 +20,5 @@
 # software solely pursuant to the terms of the relevant commercial agreement.
 
 module CrateRuby
-  VERSION = "0.0.7"
+  VERSION = "0.0.8"
```
Comment (chaudum) [inline lib/crate_ruby/version.rb]: we usually bump a version with a separate commit on master
https://github.com/crate/crate_ruby/blob/master/DEVELOP.rst#preparing-a-release

Comment (corck) [reply lib/crate_ruby/version.rb]: :+1: Reverted that commit.

**Comment by chaudum** [comment]
please squash into single commit and merge :)

**Comment by corck** [comment]
@chaudum can you enable squashing in this repo, than we can do it right here.

**Comment by chaudum** [comment]
@corck I'll have to ask Joe to enable it.

**Comment by chaudum** [comment]
@corck should be enabled now

File: lib/crate_ruby/client.rb [hunk 2]
```diff
@@ -83,13 +83,13 @@ def drop_table(table_name, blob=false)
     # List all user tables
     # @return [ResultSet]
     def show_tables
-      execute("select table_name from information_schema.tables where schema_name = 'doc'")
+      execute("select table_name from information_schema.tables where table_schema = 'doc'")
```
Comment (mfussenegger) [inline lib/crate_ruby/client.rb]: This breaks compatibility with older Crate versions - would it make sense to add a compatibility matrix in the README?

Comment (corck) [reply lib/crate_ruby/client.rb]: Probably best someone of you guys do it, as I have no clue when your change was introduced. ...

**Pull request merged**